### PR TITLE
(antu) add production host up/down alert config

### PR DESCRIPTION
**Pull request by jhoblitt**
Based on #498

### CODE REVIEW

**Review by fbegyn — CHANGES_REQUESTED** [review]
Aside from the incorrect YAML format in the `/rules` directory, this seems OK

File: fleet/lib/prometheus-alerts/rules/prometheusrule-net.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    lsst.io/rule: "true"
+  name: host
+spec:
```
[the remark this answers is not in the document]
Comment (jhoblitt) [reply fleet/lib/prometheus-alerts/rules/prometheusrule-net.yaml:8]: It wasn't intentionally committed without modification to work with the template.  What's odd is that this is actually live and working.  I will fix it.

**Review by fbegyn — APPROVED** [review]
As with some other PR, without a `receiver: ",slack,"` like label, this alerts will just be blackholed by the alertmanager. Otherwise these alerts like fine now.

**Comment by jhoblitt** [comment]
> As with some other PR, without a `receiver: ",slack,"` like label, this alerts will just be blackholed by the alertmanager. Otherwise these alerts like fine now.

At the moment, the label `prod` is being matched for sending to squadcast.

**Pull request merged**